src/rest-server/src/controllers/v2/storage.js

```diff
@@ -22,7 +8,7 @@ const storage = require('@pai/models/v2/storage');
 
 const list = asyncHandler(async (req, res) => {
   const userName = req.user.username;
-  const filterDefault = Boolean(req.query.default);
+  const filterDefault = (req.query.default === 'true');
```

Comment:
the original endpoint is to use `/api/v2/storages?default&...`

Reply:
@abuccts I think the change will break `/api/v2/storages?default&...`, is there anyone still use this query ?

Reply:
no